plugins/org.wso2.developerstudio.eclipse.artifact.synapse.api/src/org/wso2/developerstudio/eclipse/artifact/synapse/api/ui/wizard/SynapseAPICreationWizard.java

```diff
@@ -144,36 +144,26 @@ protected boolean isRequiredWorkingSet() {
 	}
 	
 	private String getTemplateContent(){
-		String content = new String();
+		StringBuilder content = new StringBuilder();
 		/*
 		 * FIXME: use template extension-point instead of hard-coding template
 		 * content
 		 */
-		content += "<api xmlns=\"";
-		content += "http://ws.apache.org/ns/synapse\"";
-		content += " context=\"";
-		content += artifactModel.getContext();
-		content += "\" name=\"";
-		content += artifactModel.getName();
-		content += "\""; 
-		if(artifactModel.getHostname()!=null && artifactModel.getHostname().length()>0){
-			content += " hostname=\"";
-			content += artifactModel.getHostname();
-			content += "\"";
+		content.append("<api xmlns=\"http://ws.apache.org/ns/synapse\" context=\"");
+		if (artifactModel.getContext().startsWith("/")) {
+			content.append(artifactModel.getContext());
```

Comment:
Shall we correct the indentation as per wso2 standard?

Reply:
Fixed in this PR : https://github.com/wso2/devstudio-tooling-esb/pull/272